Library/OcMainLib/OpenCoreUefi.c

```diff
@@ -345,7 +345,10 @@ OcReinstallProtocols (
   BOOLEAN       InstallAppleEvent;
   BOOLEAN       OverrideAppleEvent;
 
-  if (OcAudioInstallProtocols (Config->Uefi.ProtocolOverrides.AppleAudio) == NULL) {
+  if (OcAudioInstallProtocols (
```

Comment:
Better to make a variable than try to pass into an if in my opinion.

Reply:
This is the same layout as e.g. OcAppleEventInstallProtocol just below in this file - or am I misunderstanding?

Reply:
OcAppleEventInstallProtocol is also ugly :)

Reply:
Hang on, basically every single line below this works the same way ... !

Reply:
Well, we already often use `Result` variable for OcOverflow functions for exactly that purpose. But can leave here as we will use Uncrustify soon anyway.